### PR TITLE
Add Roadmap to Repo

### PR DESCRIPTION
**Pull request by DarylThayil**

### CODE REVIEW

**Review by hectormmg — APPROVED** [review]
LGTM, my only suggestion is maybe adding a library column? Or alternatively, splitting into multiple tables, one for each library. I think it may be easier for developers to look under the specific library they use for features they're looking forward to instead of reading through the deliverable titles.

**Comment by pkanher617** [comment]
Is it better to have this in it's own file or as part of the main readme?

File: roadmap.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Roadmap
+
+**Deliverable**|**Status**|**Expected Delivery**|**Description**
```
Comment (tnorling) [inline roadmap.md:3]: Can we sort the entries by Expected Delivery?

**Review by tnorling — APPROVED** [review]
We should also add a link to this document from the Readmes

File: roadmap.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+:-----:|:-----:|:-----:|:-----:
+Broker In Browser |In Progress|TBD|Application model for application nested in iframes, to support silent token acquisition where 3p cookies are blocked. 
+Msal Node Public Preview|In Progress|October 2020|Our Msal Node library is currently in private preview.  Public preview should include full Public Client and Confidential Client support.  When released, we will be commiting to supporting this library at a production level, resolving bugs in a timely mananer.
```
Comment (jo-arroyo) [inline roadmap.md:6]:
```suggestion
Msal Node Public Preview|In Progress|October 2020|Our Msal Node library is currently in private preview.  Public preview should include full Public Client and Confidential Client support.  When released, we will be committing to supporting this library at a production level, resolving bugs in a timely manner. 
```

File: roadmap.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Roadmap
+
+**Deliverable**|**Status**|**Expected Delivery**|**Description**
+:-----:|:-----:|:-----:|:-----:
+Broker In Browser |In Progress|TBD|Application model for application nested in iframes, to support silent token acquisition where 3p cookies are blocked. 
+Msal Node Public Preview|In Progress|October 2020|Our Msal Node library is currently in private preview.  Public preview should include full Public Client and Confidential Client support.  When released, we will be commiting to supporting this library at a production level, resolving bugs in a timely mananer. 
+SN + I Support In Msal Node |Not Started|December 2020|Explanation of the flow from our python library: https://github.com/AzureAD/microsoft-authentication-library-for-python/issues/60
+Refresh Token Proof of Possesion [Browser]|Not started|December 2020|Proof of Possesion is a feature that enables cryptographically locking a token to a browser.  This feature has already been shipped for Access Tokens.
```
Comment (jo-arroyo) [inline roadmap.md:8]:
```suggestion
Refresh Token Proof of Possession [Browser]|Not started|December 2020|Proof of Possession is a feature that enables cryptographically locking a token to a browser.  This feature has already been shipped for Access Tokens.
```

File: roadmap.md (hunk 1 above)
Comment (jmckennon) [reply roadmap.md:3]: +1

File: roadmap.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+Tenant Profile Support |Not Started|December 2020|Support for tenant profiles in our cache and account management
+Msal Angular 2.0 [Support for Auth Code Flow]|In Progress|December 2020|Upgrading our msal angular library to use the msal-browser lib we recently GA'd to support the Auth Code Flow in the browser.  Helps alleviate some issues caused by blocked 3p cookies. 
+Msal React Private Preview|In Progress|November 2020|Release our Msal React wrapper preview, using auth code flow in browser. 
+Msal React Public Preview|Not Started|TBD|Once we are ready to support out msal react library in production we will move to public preview. 
+Msal Node GA|Not Started|January 2021|After public preview, we will work to resolve issues and stabilize the library to be able to announce General Availibilty
```
Comment (jo-arroyo) [inline roadmap.md:13]:
```suggestion
Msal Node GA|Not Started|January 2021|After public preview, we will work to resolve issues and stabilize the library to be able to announce General Availability. 
```

File: roadmap.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+SN + I Support In Msal Node |Not Started|December 2020|Explanation of the flow from our python library: https://github.com/AzureAD/microsoft-authentication-library-for-python/issues/60
+Refresh Token Proof of Possesion [Browser]|Not started|December 2020|Proof of Possesion is a feature that enables cryptographically locking a token to a browser.  This feature has already been shipped for Access Tokens.
+Tenant Profile Support |Not Started|December 2020|Support for tenant profiles in our cache and account management
+Msal Angular 2.0 [Support for Auth Code Flow]|In Progress|December 2020|Upgrading our msal angular library to use the msal-browser lib we recently GA'd to support the Auth Code Flow in the browser.  Helps alleviate some issues caused by blocked 3p cookies. 
+Msal React Private Preview|In Progress|November 2020|Release our Msal React wrapper preview, using auth code flow in browser.
```
Comment (jo-arroyo) [inline roadmap.md:11]:
```suggestion
Msal React Private Preview|In Progress|November 2020|Release our Msal React wrapper preview, using Auth Code Flow in browser. 
```

File: roadmap.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+SN + I Support In Msal Node |Not Started|December 2020|Explanation of the flow from our python library: https://github.com/AzureAD/microsoft-authentication-library-for-python/issues/60
+Refresh Token Proof of Possesion [Browser]|Not started|December 2020|Proof of Possesion is a feature that enables cryptographically locking a token to a browser.  This feature has already been shipped for Access Tokens.
+Tenant Profile Support |Not Started|December 2020|Support for tenant profiles in our cache and account management
+Msal Angular 2.0 [Support for Auth Code Flow]|In Progress|December 2020|Upgrading our msal angular library to use the msal-browser lib we recently GA'd to support the Auth Code Flow in the browser.  Helps alleviate some issues caused by blocked 3p cookies. 
+Msal React Private Preview|In Progress|November 2020|Release our Msal React wrapper preview, using auth code flow in browser. 
+Msal React Public Preview|Not Started|TBD|Once we are ready to support out msal react library in production we will move to public preview.
```
Comment (jo-arroyo) [inline roadmap.md:12]: Out vs. our?
```suggestion
Msal React Public Preview|Not Started|TBD|Once we are ready to support our msal react library in production we will move to public preview. 
```

File: roadmap.md [hunk 7]
```diff
@@ -0,0 +1,14 @@
+# Roadmap
+
+**Deliverable**|**Status**|**Expected Delivery**|**Description**
+:-----:|:-----:|:-----:|:-----:
+Broker In Browser |In Progress|TBD|Application model for application nested in iframes, to support silent token acquisition where 3p cookies are blocked. 
+Msal Node Public Preview|In Progress|October 2020|Our Msal Node library is currently in private preview.  Public preview should include full Public Client and Confidential Client support.  When released, we will be commiting to supporting this library at a production level, resolving bugs in a timely mananer. 
+SN + I Support In Msal Node |Not Started|December 2020|Explanation of the flow from our python library: https://github.com/AzureAD/microsoft-authentication-library-for-python/issues/60
+Refresh Token Proof of Possesion [Browser]|Not started|December 2020|Proof of Possesion is a feature that enables cryptographically locking a token to a browser.  This feature has already been shipped for Access Tokens.
+Tenant Profile Support |Not Started|December 2020|Support for tenant profiles in our cache and account management
```
Comment (jo-arroyo) [inline roadmap.md:9]:
```suggestion
Tenant Profile Support |Not Started|December 2020|Support for tenant profiles in our cache and account management.
```

File: roadmap.md [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+Msal Angular 2.0 [Support for Auth Code Flow]|In Progress|December 2020|Upgrading our msal angular library to use the msal-browser lib we recently GA'd to support the Auth Code Flow in the browser.  Helps alleviate some issues caused by blocked 3p cookies.
```
Comment (jo-arroyo) [inline roadmap.md:10]:
```suggestion
Msal Angular 2.0 [Support for Auth Code Flow]|In Progress|December 2020|Upgrading our Msal Angular library to use the Msal Browser lib we recently GA'd to support the Auth Code Flow in the browser.  Helps alleviate some issues caused by blocked 3p cookies. 
```

**Review by jo-arroyo — APPROVED** [review]
Just a few nits

**Comment by jasonnutter** [comment]
> Is it better to have this in its own file or as part of the main readme?

Big fan of separate file.

File: roadmap.md [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+Msal Node Public Preview|In Progress|October 2020|Our Msal Node library is currently in private preview.  Public preview should include full Public Client and Confidential Client support.  When released, we will be committing to supporting this library at a production level, resolving bugs in a timely manner. 
+SN + I Support In Msal Node |Not Started|December 2020|Explanation of the flow from our python library: https://github.com/AzureAD/microsoft-authentication-library-for-python/issues/60
+Refresh Token Proof of Possession [Browser]|Not started|December 2020|Proof of Possession is a feature that enables cryptographically locking a token to a browser.  This feature has already been shipped for Access Tokens.
+Tenant Profile Support |Not Started|December 2020|Support for tenant profiles in our cache and account management.
+Msal Angular 2.0 [Support for Auth Code Flow]|In Progress|December 2020|Upgrading our Msal Angular library to use the Msal Browser lib we recently GA'd to support the Auth Code Flow in the browser.  Helps alleviate some issues caused by blocked 3p cookies.
```
Comment (jasonnutter) [inline roadmap.md:10]: Can we split this into public/private preview?

**Review by jasonnutter — CHANGES_REQUESTED** [review]

File: roadmap.md [hunk 10]
```diff
@@ -0,0 +1,14 @@
+# Roadmap
+
+**Deliverable**|**Status**|**Expected Delivery**|**Description**
+:-----:|:-----:|:-----:|:-----:
+Broker In Browser |In Progress|TBD|Application model for application nested in iframes, to support silent token acquisition where 3p cookies are blocked. 
+Msal Node Public Preview|In Progress|October 2020|Our Msal Node library is currently in private preview.  Public preview should include full Public Client and Confidential Client support.  When released, we will be committing to supporting this library at a production level, resolving bugs in a timely manner.
```
Comment (sameerag) [inline roadmap.md:6]: Link to node read me in github?

**Review by sameerag — APPROVED** [review]

File: roadmap.md (hunk 1 above)
Comment (DarylThayil) [reply roadmap.md:3]: will do

File: roadmap.md (hunk 9 above)
Comment (DarylThayil) [reply roadmap.md:10]: I am curious what difference we see. Honestly my hope is that we can ship the whole thing at the end of the quarter and potential have some betas ready to test before hand.

Comment (sameerag) [reply roadmap.md:10]: I still think we should split it to set customer expectations!

**Review by jasonnutter — APPROVED** [review]

**Comment by jasonnutter** [comment]
Looks good, can you also update the existing link in teh README to point to this?

**Pull request merged**